lighthouse/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub use account_manager;
+pub use beacon_node;
+pub use boot_node;
+pub use database_manager;
+pub use environment;
+pub use eth2_network_config;
+pub use lighthouse_network;
+pub use lighthouse_version;
+pub use slasher;
+pub use task_executor;
+pub use types as eth2_types;
```

Comment:
eth2 is deprecated, we could probably just call this `types` and still avoid confusion

Reply:
Addressed in [`fee55d3`](https://github.com/sigp/lighthouse/pull/5875/commits/fee55d35f60a1108d4e90a7dd5ea68383351a062).